Userland/Libraries/LibC/stdio.cpp

```diff
@@ -402,9 +402,10 @@ size_t FILE::Buffer::buffered_size() const
 
 const u8* FILE::Buffer::begin_dequeue(size_t& available_size) const
 {
-    if (m_ungotten) {
-        available_size = 1;
-        return &m_unget_buffer;
+    if (m_ungotten != 0u) {
+        auto available_bytes = count_trailing_zeroes_32(m_ungotten) + 1;
```

Comment:
```suggestion
        auto available_bytes = count_trailing_zeroes(m_ungotten) + 1;
```

We have a generic version for this; the `_32` suffixed variant will be removed in #11322

Reply:
No such luck.
```
/home/Test/Workspace/serenity/Userland/Libraries/LibC/stdio.cpp:430:32: error: 'count_trailing_zeroes' was not declared in this scope; did you mean 'count_trailing_zeroes_32'?
  430 |         auto available_bytes = count_trailing_zeroes(m_ungotten);
      |                                ^~~~~~~~~~~~~~~~~~~~~
      |                                count_trailing_zeroes_32
```

Reply:
Ah, they're in a separate header.